docs/getting-started-guides/kubeadm.md

```diff
@@ -269,7 +269,7 @@ It takes several minutes to download and start all the containers, watch the out
 
 Then go to the IP address of your cluster's master node in your browser, and specify the given port.
 So for example, `http://<master_ip>:<port>`.
-In the example above, this was `31869`, but it is a different port for you.
+In the example above, this was `30001`, but it is a different port for you.
```

Comment:
Yes, that [is correct](https://github.com/microservices-demo/microservices-demo/blob/9cfe9cae3bbeab6d79aa68f9f6c7f842e355b555/deploy/kubernetes/complete-demo.yaml#L241), thanks for fixing this!